src/main/java/frc/robot/Constants.java

```diff
@@ -147,4 +164,26 @@ public static final class AutoConstants { //TODO: The below constants are used i
             new TrapezoidProfile.Constraints(
                 kMaxAngularSpeedRadiansPerSecond, kMaxAngularSpeedRadiansPerSecondSquared);
     }
+
+    public static final class Positions {
```

Comment:
Maybe name this class a little more specifically, `Constants.Positions` is a little vague`